release-notes/opendistro-for-elasticsearch-alerting.release-notes-1.10.1.2.md

```diff
@@ -2,6 +2,9 @@
 
```

Comment:
The top line needs to be changed to `1.10.1.2`

Reply:
Good catch, updated.